tests/ui/never_loop.stderr

```diff
@@ -75,6 +75,11 @@ LL | |             _ => return,
 LL | |         }
 LL | |     }
    | |_____^
+   |
+help: if you need the first element of the iterator, try writing
+   |
+LL |     if let Some(x) = (0..10).into_iter().next() {
```

Comment:
Hmm this should be `(0..10).next()`. Sorry I didn't realize this till now - we have a `make_iterator_snippet` util that should handle all three cases.